tests/Resources/ti.ui.navigationwindow.test.js

```diff
@@ -215,19 +284,19 @@ describe.windowsMissing('Titanium.UI.NavigationWindow', function () {
 });
 
 describe('Titanium.UI.Window', function () {
-	var nav;
+	let nav;
 
 	this.timeout(10000);
 
-	afterEach(function () {
-		if (nav !== null) {
-			nav.close();
+	afterEach(done => {
+		if (nav) {
+			nav.close().then(() => done()).catch(e => done());
```

Comment:
<!--
  0 failure: 
  3 warning:  tests/Resources/t..., tests/Resources/t..., tests/Resources/t...
  
  
  DangerID: danger-id-Danger;
  File: tests/Resources/ti.ui.navigationwindow.test.js;
  Line: 293;
-->

- :warning: tests/Resources/ti.ui.navigationwindow.test.js line 293 – 'e' is defined but never used. Allowed unused args must match /^_.+/u. (no-unused-vars)
- :warning: tests/Resources/ti.ui.navigationwindow.test.js line 293 – Avoid calling back inside of a promise. (promise/no-callback-in-promise)
- :warning: tests/Resources/ti.ui.navigationwindow.test.js line 293 – Avoid calling back inside of a promise. (promise/no-callback-in-promise)